CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,50 @@
+# Code of Conduct
+
+Contact: a moderator ([see below](#moderators)), or a member of the WPT community that you feel you can trust.
+
+## Conduct
+
+* We are committed to providing a friendly, safe, and welcoming environment for all,
+ regardless of level of experience, gender, gender identity and expression, sexual orientation,
+ disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other similar characteristic.
+ * Please avoid using overtly sexual nicknames or other nicknames that might detract from a friendly, safe and welcoming environment for all.
+ * Please be kind and courteous. There's no need to be mean or rude.
+ * Respect that people have differences of opinion and that every design or implementation
+ choice carries a trade-off and numerous costs. There is seldom a right answer.
+ * Please keep unstructured critique to a minimum. If you have solid ideas you want to experiment with, make a fork and see how it works.
+ * We will exclude you from interaction if you insult, demean or harass anyone. That is not welcome behavior. We interpret the term "harassment" as including the definition in the [Citizen Code of Conduct](http://citizencodeofconduct.org/); if you have any lack of clarity about what might be included in that concept, please read their definition. In
+ particular, we don't tolerate behavior that excludes people in socially marginalized groups.
+ * Private harassment is also unacceptable. No matter who you are, if you feel you have been or are being harassed or made uncomfortable by a community member, please contact one of the moderators immediately. Whether you're a regular contributor or a newcomer, we care about making this community a safe place for you and we've got your back.
+ * Likewise any spamming, trolling, flaming, baiting, or other attention-stealing behaviour
+ is not welcome.
+
+## Moderation
+
+These are the policies for upholding our community’s standards of conduct. If you feel that a thread needs moderation, please contact one of [the moderators](#moderators).
+
+* Remarks that violate this code of conduct, including hateful, hurtful, oppressive, or exclusionary remarks, are not allowed. (Cursing is allowed, but never targeting another user, and never in a hateful manner.)
+* Remarks that moderators find inappropriate, whether listed in the code of conduct or not, are also not allowed.
+* Moderators will first respond to such remarks with a warning.
```

Comment:
I don't know if this is covered elsewhere, but I think moderators should have discretion to skip the warning step for clearly abusive or extreme behavior, especially from people who may not even be community members. Warnings are appropriate for established community members, or new members making an honest mistake, but I think that drive-by trolling or off-topic spam could justify an immediate kick/ban with no prior warning. (This could still be appealed as noted below.)

Reply:
Thanks Klaus; I've made sure to note that in the proposed new CoC based on the contributors covenant (https://github.com/web-platform-tests/wpt/pull/23762), but if we reject that one and proceed with this one I will also edit it accordingly.